### ISSUE TITLE
(#211) Build and theme jobs landing page / search

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/alnap#49]
### Done criteria
- [x] **Search with filters for:** 
   - [x] Text search
   - [x] Organisation
   - [x] Type (Taxonomy: Consultancy, ToR, Job etc)
   - [x] Location (Country taxonomy)
   - [x] Tags (Keywords)
   - [ ] Role (title?)
   - [x] ISSUE: Responsive filters

- [x] **Buttons**
  - [x] SCOPING: How to add an "Add a job" button with special permissions
  - [x] Create link/buttons for: Add a job

- [x] **View toggle**
  - [x] Switch for grid/list
  - [x] grid view icon highlight

- [x] **Card content for job listing**
  - [x] Logo 
  - [x] Title 
  - [x] Agency / member 
  - [x] Location 
  - [x] Deadline

- [x] **combine filters for sort/order by**
  - [x]  Date added
  - [x]  Deadline date

---
### Designs
- [Jobs landing](https://invis.io/F3C0I5GAX)

---
### ALNAP to check
http://alnap-dev.mrmdev.co.uk/jobs
- [x] Page looks like design
- [x] Correct filters appear
- [x] Search works
- [x] Sort works
- [x] Grid/list toggle works

--------
From OneNote:
- Related content to offer visitors other visual cues to entice them to explore. Related resources / information how to apply / general information on the sector?

**Comment by dave-manta** [comment]
@cutlern we need a taxonomy for job type.

**Comment by cutlern** [comment]
@dave-manta I've added it to the content model, and told dilwar (who's currently building Job CT)

**Comment by dave-manta** [comment]
@cutlern could you check the other search criteria above and make sure they are covered in the content type. Many thanks for passing onto Dilwar.

**Comment by cutlern** [comment]
@dave-manta Type, organisation and member are already included. Location is currently a text field,  do you think it should be a taxonomy like the country taxonomy? Will the location need to show cities or just countries?

**Comment by dave-manta** [comment]
@cutlern I think location should be a taxonomy, listing all countries in the World.

**Comment by cutlern** [comment]
@dave-manta Okie, added to content model and dilwar is going to change it

**Comment by dave-manta** [comment]
Thanks @cutlern. Top comms :)

**Comment by cutlern** [comment]
Add a job button/link will need to wait until how we let members add content has been scoped: https://app.zenhub.com/workspace/o/mantaraymedia/alnap/issues/244

**Comment by Maria-alnap** [comment]
@nadia847 I've only ticked the last 2 as there's some issues with the others:
- Because the civi name for the UK is ridiculously long, that messes up the card (see the job I added just now). Not sure how to go about this but many jobs will be from the UK so it's something we'll have to deal with.  Maybe just show city in the card?
- On the filters, we'll have more than consultancies or ToRs there (which are the only options showing up under type at the moment. I assume this will get populated further as we have other jobs?)
- I just tried searching for "research assistant", which is the job I just added, but the search didn't work

**Comment by nadia847** [comment]
@Maria-alnap 

- Length of country/city name should no longer break layout. I prefer having the full City and Country there and if I had to choose between the two would say Country is more important. I've shortened it anyway though because UKGBNI is just nuts.

- I've also changed the order to City, County because that is surely a global standard?

- Yes, as you add more jobs, the filters should have more selectable options!

- Indexing of content isn't instantaneous, it happens every time a script runs which we tend to run a limited times a day. But if you look now though, it's there :o)
